.github/workflows/update-check.yml

```diff
@@ -0,0 +1,63 @@
+name: Check for ANYK update
+
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: 'master'
+
+      - name: Download ANYK repository index
+        run: curl -L https://nav.gov.hu/abev/abev_new -o anyk.xml
+
+      - name: Get latest ANYK version number
+        id: latestanykversion
+        uses: Teko012/get-xml-info_fork_mavrosxristoforos@66d740a01d74a260037146b6cbf252d272f066f9
```

Comment:
Was there a problem with the original action?

Reply:
Yes, I have an [open PR](https://github.com/mavrosxristoforos/get-xml-info/pull/12) with the original creator, we can move to that if it get's merged. Basically Github is deprecating two things, and that needs to be fixed.